### PR TITLE
Dw 305 tabs component

### PR DESCRIPTION
**Pull request by GusBaamonde**
- add the html tabs file
- add the style for the component
- add the link in the navigation bar

tabs-component

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

**Comment by mdirago** [comment]
Excellent Gus!

File: assets/js/webPushMain.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+
+if (window.location.pathname === '/notification-poc.html') {
```
Comment (cbernat) [inline assets/js/webPushMain.js:2]: ❤️

File: assets/js/webPushMain.js [hunk 2]
```diff
@@ -26,21 +28,26 @@ var UI = {
...
-});
 
-UI.notificationButtonInt.addEventListener('click', function () {
-  if (isSubscribed) {
-    sendRequestNotification('http://appint.fromdoppler.net:5001');
-  } else {
-    console.log('Web push: No esta suscripto a notificaciones');
-  }
-});
+if (!!UI.notificationButton) {
```
Comment (cbernat) [inline assets/js/webPushMain.js:32]: I think you don't need this anymore 😄

File: assets/scss/modules/_tabs.scss [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+.wrapper-tabs {
+  margin: 10px $dp-spaces--lv0 $dp-spaces--lv9;
+  position: relative;
+  width: 100%;
+}
+
+.bg-tabs {
+  position: absolute;
+  border-top: 1px solid #ccc;
```
Comment (cbernat) [inline assets/scss/modules/_tabs.scss:19]: Please chech to use color variables here for `#ccc` and `#eaeaea`

File: assets/scss/templates/_guidelines.scss [hunk 4]
```diff
@@ -212,7 +145,7 @@ body {
 .dpsg-main-wrap {
   display: flex;
   flex-direction: column;
-  font-family: $dp-font-family-proximanova;
+  font-family: Arial, sans-serif;
```
Comment (cbernat) [inline assets/scss/templates/_guidelines.scss:148]: How about adding these font types as a variable too? maybe we reuse them somewhere else.

**Comment by cbernat** [comment]
Very nice! https://cdn.fromdoppler.com/doppler-ui-library/build856/tabs-component.html

**Comment by andresmoschini** [comment]
Muy bueno!

File: assets/templates/tabs-component.html [hunk 5]
```diff
@@ -0,0 +1,749 @@
...
+            <p>
+              Los tabs nos van a permitir crear una interfaz de usuario basada
+              en pestañas, donde, de una forma muy intuitiva, podemos ofrecer al
+              usuario diferentes vistas, que son seleccionadas al pulsar una de
+              las pestañas que se muestran a continuación.
+            </p>
+            <hr class="dpsg-break" />
+            <div class="dp-container">
+              <div class="dp-rowflex">
+                <nav class="wrapper-tabs">
```
Comment (cbernat) [inline assets/templates/tabs-component.html:86]: Perhaps here you could think of a more cohesive way for the naming of the classes, for example:
``` 
<nav class="tab-menu--wrapper">
      <ul class="tab-menu">
         <li class="tab-menu--item">
...
```
It gives you an idea of this tab as a component, tab-menu, which has many parts: a wrapper and items. 
I'm not so sure about `tab-menu`, perhaps another name would be ok, what do you think?

File: assets/scss/templates/_guidelines.scss (hunk 4 above)
Comment (GusBaamonde) [reply assets/scss/templates/_guidelines.scss:148]: _$ dp-font-family: Arial, sans-serif! default;
$ dp-font-family-proximanova: 'proxima-nova', Helvetica, Arial, sans-serif! default;
$ dp-font-family-handofsean: 'hand-of-sean';_

**If we have them I forgot to put it**

File: assets/templates/tabs-component.html (hunk 5 above)
Comment (GusBaamonde) [reply assets/templates/tabs-component.html:86]: ok, makes sense i check it

**Pull request merged**